app/assets/stylesheets/components/_card_home_page.scss

```diff
@@ -1,9 +1,10 @@
 .credit-card {
   overflow: hidden;
-  width: 600px;
+  width: 650px;
```

Comment:
This fills the space that the cards had to the right in screens larger than 1400px

Reply:
So does that mean in smaller screens it just resizes to 600 px or just has 650px and thus will take the space in larger screens?

Reply:
This is actually to fill the column in larger screens. It makes this:
<img width="1329" alt="Screen Shot 2022-11-04 at 1 12 01 PM" src="https://user-images.githubusercontent.com/108884517/200035784-99560c5a-5094-47a1-9fc1-2302411eb37c.png">
Into this:
<img width="1356" alt="Screen Shot 2022-11-04 at 1 12 56 PM" src="https://user-images.githubusercontent.com/108884517/200035826-1203cbe1-af1f-4d7c-a319-fb0c35c5c778.png">

Then we add the `max-width: 100%;` a couple of lines below to make it responsive and never let it get out of the container (the column). Otherwise it would always be 650 px even if it's in a mobile device and it would create ugly horizontal scrolls.